docs/_includes/checkbox.html

```diff
@@ -177,10 +177,10 @@ <h3 id="checkbox-group">Grouped Checkboxes</h3>
                         <input class="checkbox__control" id="group-checkbox-3" type="checkbox" value="3" name="checkbox-group" />
                         <span class="checkbox__icon" hidden>
                             <svg class="checkbox__unchecked" focusable="false" height="18" width="18" aria-hidden="true">
-                                {% include symbol.html name="checkbox-unchecked" %}
+                                {% include symbol.html name="web-checkbox-unselected-18" %}
```

Comment:
I still don't like these `web` names. I suppose we can change these later.

Reply:
I did speak with design, we can remove them. Let me ask in chat what we should name them.

Reply:
Ok, should be resolved.